src/crimson/net/SocketConnection.h

```diff
@@ -58,9 +58,9 @@ class SocketConnection : public Connection {
   bool update_rx_seq(seq_num_t seq);
 
   // messages to be resent after connection gets reset
-  std::queue<MessageRef> out_q;
+  std::deque<MessageRef> out_q;
```

Comment:
In the future we could take a look on Seastar's `chunked_fifo` for that.

Reply:
logged as https://github.com/ceph/ceph/projects/2#card-20857269